src/backends/Makefile.am

```diff
@@ -1 +0,0 @@
-SUBDIRS= interflop-ieee interflop-mca interflop-mca-int interflop-cancellation interflop-bitmask interflop-vprec
```

Comment:
Why is this removed ?

Reply:
In the previous setup, the backends were installed separately during the execution of the `install-interflop.sh` script. However,  I think it's more logical to handle the backend installation directly within verificarlo itself.

Now, each backend comes with its own configure script. These scripts are automatically executed as part of Verificarlo's main configure process. Specifically, the `ax_interflop_backends.m4` script located in `src/interflop-stdlib/m4/`  scans the `src/backends` directory to identify all available backends and then populates the `INTERFLOP_BACKENDS` variable with this information. The backends are dynamically recognized and included without the need to manually modify the SUBDIRS variable. This change simplifies the process of adding new backends, as they are now automatically detected and integrated during configuration.

Reply:
:+1: